backend/default.nix

```
@@ -1,23 +1,25 @@
 {
-  perSystem = { self', pkgs, lib, ... }:
+  perSystem = { config, self', pkgs, lib, ... }:
     let
       src = lib.sourceFilesBySuffices ./. [ ".rs" ".toml" "Cargo.lock" ];
       inherit (lib.importTOML (src + "/Cargo.toml")) package;
     in
     {
-      packages = {
-        ${package.name} = pkgs.rustPlatform.buildRustPackage {
-          pname = package.name;
-          inherit (package) version;
-          inherit src;
-          cargoLock.lockFile = (src + "/Cargo.lock");
-        };
-        default = self'.packages.${package.name};
-      };
-
+      dream2nix.inputs."backend" = {
+	source = src;
+	projects."backend" = { name, ... }: {
+	  inherit name;
+	  subsystem = "rust";
+	  translator = "cargo-lock";
+	};
+      }; 
+      packages = config.dream2nix.outputs."backend".packages;
       devShells.backend = pkgs.mkShell {
         inherit (package) name;
-        inputsFrom = [ self'.packages.${package.name} ];
+        inputsFrom = [ 
+	  config.dream2nix."backend".devShells.${package.name}
```

CI is failing. This should hopefully fix it:

```suggestion
	  config.dream2nix.outputs."backend".devShells.${package.name}
```